canonical-web-and-design/vanilla-framework
# Make examples fully work again in Flask app

## Done

- Added back missing utilities and templates examples
- Made all examples back into .html files without frontmatter
- Add jinja template blocks to examples
- Add standalone CSS to relevant examples
- Add back separate page for standalone examples

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2799.run.demo.haus/)
- Go to [examples page](https://vanilla-framework-canonical-web-and-design-pr-2799.run.demo.haus/examples)
- Make sure list of the examples has all the examples (as on live site)
- Check if examples work as expected
- Go to [standalone examples page](https://vanilla-framework-canonical-web-and-design-pr-2799.run.demo.haus/examples?standalone) (link on the bottom of examples page)
- Make sure list of standalone examples has same examples as live site (base and patterns)
- Make sure standalone examples work as expected
